packages/components/organisms/f-checkout/src/components/UserNote.vue

```diff
@@ -1,30 +1,31 @@
 <template>
     <div :class="$style['c-userNote']">
-        <label
-            for="userNote"
-            :class="$style['c-userNote-title']">
-            {{ $t('userNote.title') }}
-        </label>
-        <div :class="$style['c-userNote-content']">
-            {{ $t('userNote.text') }}
-        </div>
-        <textarea
-            id="userNote"
-            name="notePlaceholder"
-            cols="30"
-            rows="10"
-            maxlength="200"
-            :class="$style['c-userNote-textArea']"
+        <form-field
+            :label-text="$t('userNote.title')"
+            input-type="textarea"
             :placeholder="$t('userNote.placeholder')"
-            @input="updateUserNote($event.target.value)" />
+            :cols="30"
+            :rows="7"
+            :maxlength="200"
+            :show-extra-label-text="true"
+            :name="notePlaceholder"
+            :has-input-description="true"
+            @input="updateUserNote($event)">
```

Comment:
Do you need to explicitly pass `$event` here?

Reply:
f-form-field component emits input event and passes only value so $event in this case is the value: https://github.com/justeat/fozzie-components/blob/e33577a7442ddf150bfa35a69c24ef2f5d9ec1f2/packages/components/atoms/f-form-field/src/components/FormField.vue#L261